Source/JavaScriptCore/wasm/WasmOMGPlan.cpp

```diff
@@ -103,15 +103,15 @@ void OMGPlan::work(CompilationEffort)
 
     if (UNLIKELY(!parseAndCompileResult)) {
         Locker locker { m_lock };
-        fail(makeString(parseAndCompileResult.error(), "when trying to tier up ", String::number(m_functionIndex)));
+        fail(makeString(parseAndCompileResult.error(), "when trying to tier up "_s, String::number(m_functionIndex)));
```

Comment:
Should say `m_functionIndex` instead of `String::number(m_functionIndex)`. Doesn’t need to be addressed in this patch, but I probably would.